eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/filter/views/EventFilterView.java

```diff
@@ -77,6 +77,7 @@
     private CategorySelectionChangeListener mCategorySelectionChangeListener;
     private LinearLayout mDropDown;
     private List<FilterItemView> mItemViews;
+    private SmartView<Boolean> mShowAllItemView;
```

Comment:
I think it would be worth creating a `SmartView<FilterState>` for this item, so that we only need to update it with `mFilterState`, a little bit less thinking and complexity in this `EventFilterView`, no need to use `true` or `false`.
The class of course would delegate to `Highlightable` and use the negated `hasSelection()`.